packages/terminal-extension/src/index.ts

```diff
@@ -296,16 +350,21 @@ export function addCommands(
     }
   });
 
-  commands.addCommand(CommandIDs.toggleTheme, {
-    label: 'Use Dark Terminal Theme',
-    caption: 'Whether to use the dark terminal theme',
-    isToggled: () => Terminal.defaultOptions.theme === 'dark',
-    execute: () => {
-      let { theme } = Terminal.defaultOptions;
-      theme = theme === 'dark' ? 'light' : 'dark';
+  commands.addCommand(CommandIDs.setTheme, {
+    label: args => {
+      const theme = args['theme'] as string;
+      const displayName = theme[0].toUpperCase() + theme.substring(1);
+      return args['isPalette']
+        ? `Use ${displayName} Terminal Theme`
+        : displayName;
+    },
+    caption: 'Set the terminal theme',
+    isToggled: args => args['theme'] === Terminal.defaultOptions.theme,
```

Comment:
I don't understand this `isToggled` condition. Shouldn't we be checking the currently set theme options, rather than the default ones?